### PR TITLE
Catch errors from search endpoint

### PR DESCRIPTION
**Pull request by mruwnik**
https://github.com/StampyAI/stampy-ui/issues/217

image

### CODE REVIEW

**Comment by mruwnik** [comment]
should the error be nicely formatted?

File: app/routes/questions/search.tsx [hunk 1]
```diff
@@ -15,6 +15,10 @@ export function search(question: string, onlyLive: boolean) {
     onlyLive ? 1 : 0
   }`
   return fetch(url).then(async (response) => {
+    if (response.status != 200) {
+      console.error(await response.text())
+      return new Response('Error while searching for questions', {status: 502})
```
Comment (mruwnik) [inline app/routes/questions/search.tsx:20]: I considered returning what the downstream server returns, but that's usually random HTML, so decided against it

Comment (Aprillion) [reply app/routes/questions/search.tsx:20]: please do - we don't have to show it in UI, but at least make it visible in network tab of the browser

`console.error` from here is going to CloudFlare, not very useful for everyday debugging 😅

Comment (Aprillion) [reply app/routes/questions/search.tsx:20]: (and no need for nice formatting of error in UI)

**Review by Aprillion — APPROVED** [review]

**Pull request merged**